lib/util/cache/repository/types.ts

```diff
@@ -28,7 +28,7 @@ export interface BranchCache {
   isModified: boolean;
   prNo: number | null;
   sha: string | null;
-  parentSha: string | null;
+  parentSha: string | null; // SHA of baseBranch
```

Comment:
use a tsdoc comment, so you see it when hover. over that fiel in vscode on any file.

Reply:
Will remove the comment for now, as the name will be changed in upcoming PR.

Reply:
but would be good to have the tsdoc anyways. so ti's clear in future what it is for, even somebody renamed it.